reverse_words/__main__.py

```diff
@@ -24,12 +25,18 @@
              " In the `sample` and `beam_search` modes a trained model is "
              " to used reverse words in the input text.")
     parser.add_argument(
-        "save_path", default="chain",
+        "save_path", default="chain", nargs='?',
```

Comment:
The default name "chain" is a remnant from the past of this  script. Could you please change it something more sensible? E.g. `reverse_words.zip`.